metsim/methods/mtclim.py

```diff
@@ -26,74 +21,61 @@ def run(forcing: pd.DataFrame, disagg=True):
     lat_idx = forcing.index.names.index('lat')
     time_idx = forcing.index.names.index('time')
     sg = solar_geom(forcing['elev'][0], forcing.index.levels[lat_idx][0])
+    sg = {'tiny_rad_fract' : sg[0],
```

Comment:
Unfortunately, we can't do that since that forces numba into object mode, which drastically slows this down.